packages/market-service/src/osmosis/osmosis.test.ts

```diff
@@ -14,7 +14,13 @@ import {
 jest.mock('axios')
 
 const mockedAxios = axios as jest.Mocked<typeof axios>
-const osmosisMarketService = new OsmosisMarketService()
+const osmosisMarketService = new OsmosisMarketService({
+  jsonRpcProviderUrl: '',
+  unchainedEthereumHttpUrl: '',
+  unchainedEthereumWsUrl: '',
+  osmosisMarketDataUrl: 'https://api-osmosis.imperator.co/',
```

Comment:
if we can pass empty strings here to satisfy the compiler and avoid making potential network requests in tests that would be ideal